opentelemetry-user-events-logs/src/logs/processor.rs

```diff
@@ -66,6 +66,7 @@ impl opentelemetry_sdk::logs::LogProcessor for Processor {
 #[derive(Debug)]
 pub struct ProcessorBuilder<'a> {
     provider_name: &'a str,
+    resource_attribute_keys: Vec<String>,
```

Comment:
Why not use a `HashSet` to begin with instead of collecting strings into a vec and then collecting the vec into `Hashset` in the exporter new method.